node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/StorageMaintainer.java

```diff
@@ -63,44 +66,94 @@ public StorageMaintainer(DockerOperations dockerOperations, ProcessExecuter proc
 
         Dimensions dimensions = new Dimensions.Builder().add("role", "docker").build();
         numberOfNodeAdminMaintenanceFails = metricReceiver.declareCounter(MetricReceiverWrapper.APPLICATION_DOCKER, dimensions, "nodes.maintenance.fails");
+        numberOfCoredumpsOnHost = metricReceiver.declareGauge(MetricReceiverWrapper.APPLICATION_DOCKER, dimensions, "nodes.coredumps");
     }
 
     public void writeMetricsConfig(ContainerName containerName, NodeSpec node) {
-        final Path yamasAgentFolder = environment.pathInNodeAdminFromPathInNode(
-                containerName, Paths.get("/etc/yamas-agent/"));
+        List<SecretAgentCheckConfig> configs = new ArrayList<>();
 
+        // vespa
         Path vespaCheckPath = environment.pathInNodeUnderVespaHome("libexec/yms/yms_check_vespa");
-        SecretAgentCheckConfig vespaSchedule = new SecretAgentCheckConfig("vespa", 60, vespaCheckPath, "all")
-                .withTag("parentHostname", environment.getParentHostHostname());
+        SecretAgentCheckConfig vespaSchedule = new SecretAgentCheckConfig("vespa", 60, vespaCheckPath, "all");
+        configs.add(annotatedCheck(node, vespaSchedule));
 
+        // vespa-health
```

Comment:
`yms_check_vespa` and `yms_check_host_life` do not work for config server

Reply:
Do you know why host_life fails on the configserver?